File: Core/YABACore/src/commonMain/kotlin/dev/subfly/yabacore/webview/YabaWebPlatform.kt
package dev.subfly.yabacore.webview

/**
 * Host UI platform passed to the web reader/editor for theming.
 */
enum class YabaWebPlatform {
    Compose,
    Darwin,
}

/**
 * Light/dark appearance hint for web components.
 */
enum class YabaWebAppearance {
    Auto,
    Light,
    Dark,
}

/**
 * Vertical scroll direction from touch gestures (toolbar visibility).
 */
enum class YabaWebScrollDirection {
    Up,
    Down,
}

File: Darwin/YABACore/WebView/Model/YabaWebPlatform.swift
//
//  YabaWebPlatform.swift
//  YABACore
//
//  Parity with Compose `YabaWebPlatform` — host surface passed to web shells for theming.
//

import Foundation

public enum YabaWebPlatform: String, Sendable, Codable {
    case compose
    case darwin
}
